CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-See this project's [releases](/../../../releases).
+See this project's [releases](https://github.com/mayurankv/Obsidian-Code-Styler/releases).
```

Comment:
While we're at it - this was a broken link (missing the repo part of the URL), so I went ahead and fixed it with an absolute link if that sounds good to you

Reply:
Btw, I also noticed that [manifest.json](https://github.com/mayurankv/Obsidian-Code-Styler/blob/99f7ea21250f7465585dd526013f6e918d167bc2/manifest.json#L9) has a broken Buy Me A Coffee link. If you want I can also go ahead and add that quick fix to this PR (or separate PR if you'd prefer).